src/settings/tools/strip-unit.scss

```diff
@@ -1,3 +1,12 @@
+// Ref: https://github.com/sass-collective/sass-collective/blob/master/packages/strip-unit/_functions.scss
```

Comment:
Could we use this dependency and use it directly instead?

Reply:
I like this idea. I'll try to set this up and will reach out for help f I get stuck 🙂.